Tools/autotest/arducopter.py

```diff
@@ -3944,6 +3944,182 @@ def test_guided_local_velocity_target(self, vx, vy, vz_up, timeout=3):
 
         self.progress("Received proper target velocity commands")
 
+    def test_guided_local_acceleration_target(self, afx, afy, afz_up, timeout=3):
```

Comment:
Let's move this test to a separate PR.  This PR is really just adding the pause/continue (pause/resume?) feature to Copter so let's limit the autotest changes to just this feature as well.